package.json

```diff
@@ -27,6 +29,7 @@
     "cssnano": "^5.0.17",
     "enzyme": "^3.11.0",
     "enzyme-to-json": "^3.5.0",
+    "file-loader": "^6.2.0",
```

Comment:
where do you use the file-loader in the code?

Reply:
File loader is used in webpack.dev.js. I was searching for a way to test the use of static images with Typescript and this is one implementation that I tried out that worked.